packages/amplify-category-auth/src/provider-utils/awscloudformation/index.ts

```diff
@@ -1,46 +1,51 @@
-// TODO enable eslint after converting to TS
-/* eslint-disable */
-const inquirer = require('inquirer');
-const _ = require('lodash');
-const { stateManager, open } = require('amplify-cli-core');
-const { ensureEnvParamManager } = require('@aws-amplify/amplify-environment-parameters');
-const { getAuthResourceName } = require('../../utils/getAuthResourceName');
-const { copyCfnTemplate, saveResourceParameters } = require('./utils/synthesize-resources');
-const {
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I'd disable no-explicit-any globally. We have `$TSAny` but that's just a cheat.

Reply:
I agree it should be disabled globally, but I would prefer to do that in a separate PR. If it's ok, I would like to keep the explicit local disable in this PR, and then disable the rule in a follow up PR.